firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -3410,6 +3410,38 @@ extension String {
         comment: "The label for the URL field when editing a bookmark")
 }
 
+// MARK: - Root Bookmarks Folder Empty State
+extension String {
+    public struct RootBookmarksFolderEmptyState {
```

Comment:
Can we do better, organizationally, than this? Is there a bookmarks folder struct we can fold under or something. This is.... not great. lol

Reply:
There's some strings for the MVP+1 feature [that were added here:](https://github.com/mozilla-mobile/firefox-ios/pull/21508/files) maybe grouping them with them/closer to them would be better.